api/dto/target.go

```diff
@@ -100,17 +100,17 @@ var (
 	}
 )
 
-type problemOfTarget struct {
+type ProblemOfTarget struct {
```

Comment:
hmm, why become struct as public?

Reply:
This struct is used in tests when compare expected and actual

Reply:
only for tests? I think better leave struct private to avoid unexpected usage and make constructor....

Reply:
i think myself that public structs are evil, but ok, lets leave, and add godoc please)